Automattic/wp-calypso
# Videos UI: Add checkmark loaded from server

This PR loads the video completion from the API and displays a checkmark accordingly and also fixes padding issues around the accordion item titles (I did that here since I had to reorder this element content).

#### Before & After checkmark

| Before  | After |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/375980/143291448-5f337ffa-a4ff-4486-b15e-ab3d610b2778.png) | ![image](https://user-images.githubusercontent.com/375980/143291402-db4cc9af-b124-41e6-9881-405cbb77bb8a.png) |
| ![image](https://user-images.githubusercontent.com/375980/143291289-c7a3746c-5988-4b68-8e6e-52026815f3d2.png) | ![image](https://user-images.githubusercontent.com/375980/143291196-7f08cd1b-60d4-48e4-a43c-e971c78a69f1.png) |

#### Before & After title padding
| Before  | Figma | After |
| ------------- | --- | ------------- |
|![image](https://user-images.githubusercontent.com/375980/143253691-6d0d0c1f-a01d-496c-b224-2e3942b66d59.png) | ![image](https://user-images.githubusercontent.com/375980/143253899-ff618491-e2ec-44e4-9e97-bf57e14f3ee6.png) | ![image](https://user-images.githubusercontent.com/375980/143290966-f2886ba2-e5eb-407d-865b-b7b4a98f3659.png)  |

#### Testing instructions
1. Apply D70545-code and  sandbox API if not merged.
2. Make sure the user you are testing with has some videos marked as complete (you can see instructions in this diff: D70238-code).
3. Verify that the viewed checkmark shows in accordance to what you have server side.
4. Verify that everything looks OK in different viewports.

Closes to https://github.com/Automattic/wp-calypso/issues/58256
